views/update/update-filing-date.html

```diff
@@ -13,6 +13,8 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    {% include "includes/list/errors.html" %}
```

Comment:
Don't we usually get this as part of the layout.html and do you know why it wasn't working as expected?

Reply:
I don't think we do, all our views we have to add this includes line too. Checking the layout.html it looks like that just imports the `govukErrorSummary` which allows us to make use of it.